src/main/java/com/testmonitor/actions/TestCases.java

```diff
@@ -234,4 +233,36 @@ public TestCase update(TestCase testCase) throws IOException {
 
         return TestCaseParser.parse(updatedTestCase);
     }
+
+    /**
+     * Move a test case to another folder
+     *
+     * @param testCaseId The test case id you want to update
+     * @param folderId The test case folder id you want to update
+     *
+     * @return The moved test case
+     */
+    public TestCase move(int testCaseId, int folderId) throws IOException {
```

Comment:
We both use testCaseFolderId and folderId, shouldn't we pick one? testCaseFolderId?